resources/assets/components/CampaignSingle/index.js

```diff
@@ -98,8 +98,13 @@ class CampaignSingle extends React.Component {
       <div className="container">
         <StatusCounter postTotals={this.props.post_totals} campaign={campaign} />
 
-        <PostFilter onChange={this.filterPosts} />
+        <h2 className="heading -emphasized">Post Filters</h2>
+        <FilterBar onSubmit={this.filterPosts}>
+          <DropdownFilter options={{accepted: 'Accepted', pending: 'Pending', rejected: 'Rejected'}} onChange={this.updateFilters} />
```

Comment:
I think I might have messed this up in my gist example, but you shouldn't have to pass an `onChange` prop here since you pass in when you render it in `FilterBar`.